RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerAnalFit.cc

```diff
@@ -133,6 +137,21 @@ EcalUncalibRecHitWorkerAnalFit::run( const edm::Event& evt,
         return true;
 }
 
+edm::ParameterSetDescription
+EcalUncalibRecHitWorkerAnalFit::getAlgoDescription() {
+
+  edm::ParameterSetDescription psd;
+//psd.addNode(edm::ParameterSet<edm::InputTag>("EBdigiCollection", edm::InputTag("ecalDigis","ebDigis")) and
+//	       edm::ParameterSet<std::string>("EEhitCollection", "EcalUncalibRecHitsEE") and
+//	       edm::ParameterSet<edm::InputTag>("EEdigiCollection", edm::InputTag("ecalDigis","eeDigis")) and
+//	       edm::ParameterSet<std::string>("algo", "EcalUncalibRecHitWorkerAnalFit") and
+//	       edm::ParameterSet<std::string>("EBhitCollection", "EcalUncalibRecHitsEB"));
+//
+  return psd;
```

Comment:
Do you really want to return an empty parameter set description? If so, you should add a comment explaining under what circumstances the lines that are commented out should be uncommented.